uploadPatchSet.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+FLOW="$1"
+ENVIRONMENT="$2"
+
+case $# in
+  2)
+    ENVIRONMENT="$2"
+    ;;
+  1)
+    ENVIRONMENT="local"
+    ;;
+  *)
+    echo "not enough arguments supplied.  You must supply the flowName to this command."
+    return 1
+    ;;
+esac    
+
+source setEnvForUpload.sh $ENVIRONMENT
+
+if [ -z $FLOW_TOKEN ] ;
+then
+	return 1
```

Comment:
Running this script with an invalid token name (e.g. fffffff) printed the following:

```
no available flow token.  You must have a /Users/tim/creds/fffffff.token file populated with an active flow access token from the server.
/Users/tim/src/ihub-developer-scripts/uploadPatchSet.sh: line 22: return: can only `return' from a function or sourced script
/Users/tim/src/ihub-developer-scripts/uploadPatchSet.sh: line 27: [: !=: unary operator expected
cat: uploadFlowResponse.txt: No such file or directory
```

Changing line 22 to `exit 1` printed the following:

`no available flow token.  You must have a /Users/tim/creds/fffffff.token file populated with an active flow access token from the server.`

Reply:
fixed